discord_bots/models.py

```diff
@@ -1261,6 +1285,57 @@ class RotationMap:
     map_votes = relationship("MapVote", cascade="all, delete-orphan")
 
 
+@mapper_registry.mapped
+@dataclass
+class Schedule:
+    """
+    Stores days and times for scheduling games up to a week in advance
+    """
+
+    __sa_dataclass_metadata_key__ = "sa"
+    __tablename__ = "schedule"
+    __table_args__ = (UniqueConstraint("day", "time"),)
+
+    day: str = field(metadata={"sa": Column(String, nullable=False)})
+    time: time = field(metadata={"sa": Column(Time, nullable=False)})
```

Comment:
I'm not sure about datetime.time, but make sure that these have a default factory that ensures UTC time. I just want to be certain that everything that gets inserted here includes the UTC tzinfo, so that comparing them later is easier.